### ISSUE TITLE
Combobox: unit test throws uncaught errors

### ISSUE THREAD

**Issue by wkeese** [ibm-js/deliteful#481]
Run http://localhost/deliteful/node_modules/intern/client.html?config=tests/intern&suites=deliteful/tests/unit/Combobox (your URL may vary) on FF.

In the console there are a bunch of errors like:

```
"Error occured in observer callback: List<.refreshRendering@http://localhost/deliteful/list/List.js:267:1
stubChain/<@http://localhost/dcl/mini.js:189:7
Invalidating<.initializeInvalidating/this._hRendering<@http://localhost/decor/Invalidating.js:30:6
Stateful.PropertyListObserver.prototype.open/this._boundCallback<@http://localhost/decor/Stateful.js:291:7
Observable.deliverChangeRecords@http://localhost/decor/Observable.js:381:5
Stateful.PropertyListObserver.prototype.deliver@http://localhost/decor/Stateful.js:305:27
Invalidating<.deliver@http://localhost/decor/Invalidating.js:60:1
List<.deliver</<@http://localhost/deliteful/list/List.js:331:5
.attachedCallback@http://localhost/delite/StoreMap.js:139:4
stubChain/<@http://localhost/dcl/mini.js:189:7
List<.attachedCallback</<@http://localhost/deliteful/list/List.js:251:5
makeAOPStub/x@http://localhost/dcl/dcl.js:34:10
._initList@http://localhost/deliteful/Combobox.js:345:5
.attachedCallback@http://localhost/deliteful/Combobox.js:334:6
makeAOPStub/x@http://localhost/dcl/dcl.js:34:10
Widget<.startup@http://localhost/delite/Widget.js:283:5
initCombobox@http://localhost/deliteful/tests/unit/Combobox.js:86:3
createMyCombobox@http://localhost/deliteful/tests/unit/Combobox.js:100:3
suite["Store.add (user's non-trackable Memory store)"]@http://localhost/deliteful/tests/unit/Combobox.js:350:12
Test.prototype.run@http://localhost/deliteful/node_modules/intern//lib/Test.js:169:14
runTest/<@http://localhost/deliteful/node_modules/intern//lib/Suite.js:237:8
signalListener@http://localhost/deliteful/node_modules/intern//node_modules/dojo/Deferred.js:37:21
Deferred/promise.then@http://localhost/deliteful/node_modules/intern//node_modules/dojo/Deferred.js:258:5
runTest@http://localhost/deliteful/node_modules/intern//lib/Suite.js:236:1
runNextTest/<@http://localhost/deliteful/node_modules/intern//lib/Suite.js:249:7
" Observable.js:383
```

**Comment by AdrianVasiliu** [comment]
Strange that those errors aren't visible even in the latest travis build, see https://github.com/ibm-js/deliteful/issues/482#issuecomment-71597613 - nor in the detailed log https://s3.amazonaws.com/archive.travis-ci.org/jobs/48391790/log.txt.

**Comment by wkeese** [comment]
Actually that's normal, because the errors happen on a separate thread (triggered from a setTimeout).  So they are only visible in the console.

**Comment by AdrianVasiliu** [comment]
Fixed via 649ef6f342cb5f236b783c0b6dcbe65810227690. 

As far as i can see, the trouble was in List, not in Combobox. I think the bug was introduced with the remove of the scrolling container of List (in bf89f6aaf83e40de2401a98c885f4df3a7d76229). Before, the code was iterating on children of the scrolling container - all of them have a renderNode property. But now it iterates over children of the root node, which happens to contain (sometimes) the loading panel (the one showing a progress indicator), which doesn't have this property.

I rephrase the title of the issue accordingly.
